src/index.ts

```diff
@@ -20,7 +20,6 @@ export interface CdktfProviderProjectOptions extends cdk.JsiiProjectOptions {
   readonly terraformProvider: string;
   readonly cdktfVersion: string;
   readonly constructsVersion: string;
-  readonly jsiiVersion?: string;
```

Comment:
this was removed because it is already declared in `JsiiProjectOptions` which JSII started to complain about